src/Trakx.Kaiko.ApiClient.Stream.Tests/Integration/SpotExchangeRateIntegrationTests.cs

```diff
@@ -12,21 +10,36 @@ public SpotExchangeRateIntegrationTests(KaikoStreamFixture fixture, ITestOutputH
     }
 
     [Theory]
-    [InlineData("btc")]
-    [InlineData("eth")]
-    public async Task Stream_should_return_prices(string symbol, string currency = "usd")
+    [InlineData(EnabledServices.SpotExchangeRate, "btc")]
+    [InlineData(EnabledServices.SpotExchangeRate, "eth")]
+    public async Task Stream_should_return_prices(bool serviceEnabled, string symbol, string currency = "usd")
     {
         var replies = await StreamAsync(symbol, currency, StatusCode.Cancelled);
-        replies.Should().BeGreaterThan(0);
+        if (serviceEnabled)
```

Comment:
[suggestion] This code block is repeated a few times. Could this be moved into a static method to make the validation and be reused?

Reply:
Thanks, moved to a protected method in the base class.
For reference, We have an issue with the ApiClient as the Kaiko subscription doesn't give us access to all their endpoints. Some integration tests worked while we had trial access to the "X Exchange Rate" streams, but those are unavailable now - hence why this new `serviceEnabled` flag, to allow us to switch tests on/off easily.